quickevent/app/plugins/Event/src/eventplugin.cpp

```diff
@@ -636,8 +637,10 @@ void EventPlugin::connectToSqlServer()
 								     "Program features will be limited.\n\n"
 								     "To connect to a database or to choose a working directory where event files can be stored, navigate to:\n "
 								     "\"File -> Connect to database\" "));
+				break;
 			}
-			break;
+			return;
```

Comment:
do not process `!connect_ok` at the end of this function if dialog is cancelled but connection to the database is present